ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/util/FileUtil.java

```diff
@@ -24,11 +34,41 @@ public static VirtualFile getDataSourceFile(Project project, DataSourceApi dataS
     public static VirtualFile getScratchFile(Project project, String fileName) throws IOException {
         return ScratchFileService.getInstance().findFile(
                 ParameterRootType.getInstance(),
-                 project.getName() + fileName,
+                project.getName() + "_" + fileName,
                 ScratchFileService.Option.create_if_missing
         );
     }
 
+    public static String getParams(VirtualFile file) {
```

Comment:
This is crazy.

Refactor to linear if statements.

Example:

```
if (stmnt) {
  return 
}

if (stmnt) {
  return
}

mainwork
return
```

Reply:
Fixed in [b481123](https://github.com/neueda/jetbrains-plugin-graph-database-support/pull/107/commits/b481123a4b4a9620b825b946d8ac783aa1d0c49f)